### DISCUSSION TITLE
New SDK release with JS

### DISCUSSION THREAD

**Discussion by bch15** [near/near-sdk-rs#884] [Q&A] [answered]
With the release of the new near skd for smart contracts with js, is it the right thing to use and master this SDK?
Will there be many differences between these two sdk?

**Comment by austinabell** [comment, marked as answer]
So the high-level goals for the projects are:
JS SDK: quick to prototype and an easier transition for web developers, shouldn't be used for any financial products in production
Rust SDK: More complete feature set, safe, and optimized use case for building production quality dApps

I would personally recommend using the Rust SDK and only use the JS one if you strongly prefer JS and want to prototype a contract

**Reply by bch15** [reply]
Thank you for the complete and logical answer.
